api/get_transaction.go

```diff
@@ -43,7 +43,7 @@ func (a *API) unsafeGetTransaction(hash common.Hash) (*dto.TransactionReceipt, e
 	}
 
 	return &dto.TransactionReceipt{
-		TransactionWithBatchDetails: *transaction,
```

Comment:
We accidentally introduced a breaking change in `hubble_getTransaction` result. The result right now looks like this:
```json
{
    "Transaction": {
        "Hash": "0x9b442316136f46247a399169aff5b9931060331f4b66971766a81b77765cfb36",
        "TxType": "TRANSFER",
        "FromStateID": 1,
        "Amount": "50",
        "Fee": "1",
        "Nonce": "0",
        "Signature": "0x285c41448af24987934c56de4488f555ac1e1b8e1d243435af669e3ddba1c21721c4e6d62ec58de52de5fa0e146313b0d2aeb971337a30f7677fb6781ce2c058",
        "ReceiveTime": 1640176032,
        "CommitmentID": {
            "BatchID": "1",
            "IndexInBatch": 0
        },
        "ErrorMessage": null,
        "ToStateID": 2
    },
    "BatchHash": "0x367614b143b5c0db42d5cfce0e65bb657bca45d5ef5b770efa21822bca269b22",
    "BatchTime": 1640176035,
    "Status": "IN_BATCH"
}
```

while it should look like this instead (see JSON RPC docs):
```json
{
    "Hash": "0x9b442316136f46247a399169aff5b9931060331f4b66971766a81b77765cfb36",
    "TxType": "TRANSFER",
    "FromStateID": 1,
    "Amount": "50",
    "Fee": "1",
    "Nonce": "0",
    "Signature": "0x19721d261934684e37c582a1cbd69eb406eb430e56c1865e0bd071d350728c5225fc666aeb79165b6b29616a1018aabf0d317d60b0feb5668e372d7810081d96",
    "ReceiveTime": 1625153276,
    "CommitmentID": {
        "BatchID": "1",
        "IndexInBatch": 0
    },
    "ErrorMessage": null,
    "ToStateID": 2,
    "BatchHash": "0xeb590ba0ce14d821caebc56514fe867521da78b46b7b78ce4810a353e619f315",
    "BatchTime": 1633692591,
    "Status": "FINALISED"
}
```
Could you fix that as part of this PR?

Reply:
Yes, of course.